### PR TITLE
Fixes example snippet in ExistentialCollection.swift

### PR DESCRIPTION
**Pull request by valeriyvan**
Fixes example snippet in ExistentialCollection.swift

### CODE REVIEW

File: stdlib/public/core/ExistentialCollection.swift [hunk 1]
```diff
@@ -51,7 +51,7 @@ public struct AnyIterator<Element> {
   ///     func digits() -> AnyIterator<String> {
   ///         let lazyStrings = (0..<10).lazy.map { String($0) }
   ///         let iterator:
-  ///             LazyMapIterator<IndexingIterator<Range<Int>>, String>
+  ///             LazyMapSequence<(Range<Int>), String>.Iterator
```
Comment (xwu) [inline stdlib/public/core/ExistentialCollection.swift:54]: Why are the parentheses required?

Comment (valeriyvan) [reply stdlib/public/core/ExistentialCollection.swift]: They are not required. My fault. That how Xcode inferred the type.

**Comment by CodaFi** [comment]
@swift-ci smoke test and merge

**Comment by CodaFi** [comment]
@swift-ci smoke test

**Review by xwu — COMMENTED** [review]
lgtm

**Comment by CodaFi** [comment]
⛵

**Pull request merged**